addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2560,7 +2574,11 @@ const RangeUserValueWidget = UnitUserValueWidget.extend({
         const inputValue = possibleValues.length > 1 ? possibleValues.indexOf(value) : this._value;
         this.input.value = inputValue;
         if (this.displayValue) {
-            this.outputEl.value = inputValue;
+            let c = Number(inputValue);
+            if (this.exponential && c >= 0) {
+                c++;
+            }
+            this.outputEl.value = c.toFixed(this.fixedDigits);
```

Comment:
Could you explain to me what is the point of doing this and also add a comment.
If it is only used when `this.exponential` is true then you should add an "if" somewhere to avoid this code for normal cases.